functional-tests/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "functional-tests",
+  "version": "1.0.0",
+  "description": "Node functional tests",
+  "main": "index.ts",
+  "type": "module",
+  "scripts": {
+    "test": "mocha test/**/*.test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoldenGateGGX/golden-gate.git"
+  },
+  "keywords": [
+    "autotests",
+    "mocha",
+    "functional"
+  ],
+  "author": "Maksym Nyrka",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/GoldenGateGGX/golden-gate/issues"
```

Comment:
As we renamed our github, probably it makes sense to use ggxchain instead og GoldenGateGGX

Just a minor note

Reply:
Thanks, will update it.